conf/manage.py

```diff
@@ -1,4 +1,4 @@
-#!__FINALPATH__/venv/bin/python
+#!/usr/bin/env python3
```

Comment:
Hmm is that really the thing we want though ? Doesn't this allow to call the manage.py script without explicitly sourcing the venv ?

Reply:
Hmm yeah, i didn't think about people manually running it. I'll revert.